src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -47,16 +55,19 @@ protected function describeRouteCollection(RouteCollection $routes, array $optio
 
         $tableRows = [];
         foreach ($routes->all() as $name => $route) {
+            $controller = $route->getDefault('_controller');
+            list ($controllerClass, $controllerMethod) = explode('::', $controller);
```

Comment:
can be done inside transformControllerToLink - should be more robust: not all controllers have `::` inside - eg invokable ones - and controllers can be arrays, or even regular callables.